### ISSUE TITLE
How to training with RTX2080Ti * 4 GPUs to get the 79.61 mIoU results?

### ISSUE THREAD

**Issue by wqtwjt1996** [open-mmlab/mmsegmentation#332]
Hello, I trained a model with the code, and I got 79.18% mIoU using RTX2080Ti * 4. 
The config file that I used is:
**configs/deeplabv3plus/deeplabv3plus_r50-d8_512x1024_40k_cityscapes.py**

However, the reported number by MMSegmentation using the same config is 79.61% mIoU. 
![截屏2021-01-04 下午8 33 59 2](https://user-images.githubusercontent.com/22970429/103711927-7d388480-4f86-11eb-8c46-9f1ac09e1079.png)
Is 0.43% a big difference? 

I checked the log file that MMSegmentation used Tesla V100 * 4 when training (with 32 GB memory for each GPU). 
`{"env_info": "sys.platform: linux\nPython: 3.6.10 |Anaconda, Inc.| (default, May  8 2020, 02:54:21) [GCC 7.3.0]\nCUDA available: True\nCUDA_HOME: /mnt/lustre/share/cuda-10.1\nNVCC: Cuda compilation tools, release 10.1, V10.1.168\nGPU 0,1,2,3: Tesla V100-SXM2-32GB\nGCC: gcc (GCC) 5.4.0\nPyTorch: 1.5.0\nPyTorch compiling details: PyTorch built with:\n  - GCC 7.3\n  - C++ Version: 201402\n  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications\n  - Intel(R) MKL-DNN v0.21.1 (Git Hash 7d2fd500bc78936d1d648ca713b901012f470dbc)\n  - OpenMP 201511 (a.k.a. OpenMP 4.5)\n  - NNPACK is enabled\n  - CPU capability usage: AVX2\n  - CUDA Runtime 10.1\n  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37\n  - CuDNN 7.6.3\n  - Magma 2.5.2\n  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_INTERNAL_THREADPOOL_IMPL -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, \n\nTorchVision: 0.6.0a0+82fd1c8\nOpenCV: 4.2.0\nMMCV: 0.5.5\nMMSegmentation: 1.1.0+074c5a3\nMMSegmentation Compiler: GCC 5.4\nMMSegmentation CUDA Compiler: 10.1", "seed": 0, "exp_name": "deeplabv3plus_r50_512x1024_40ki_cityscapes.py"}`

So for RTX2080Ti * 4, how do I get the 79.61 mIoU results?

My OS version:
`Linux version 5.4.0-58-generic (buildd@lcy01-amd64-004) (gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)) #64-Ubuntu SMP Wed Dec 9 08:16:25 UTC 2020`

Here is my conda env for running the code:
`python                    3.7.9                h7579374_0 `
`pytorch                   1.6.0           py3.7_cuda10.2.89_cudnn7.6.5_0    pytorch`
`torchvision               0.7.0                py37_cu102    pytorch`
`mmcv-full                 1.2.5                    pypi_0    pypi`
`mmsegmentation            0.10.0                    dev_0    <develop>`
`cityscapesscripts         2.1.7                    pypi_0    pypi`
`cudatoolkit               10.2.89              hfd86e86_1  `

Thanks!

**Comment by xvjiarui** [comment]
Hi @wqtwjt1996 
0.43% is a reasonable variance for it.
